spec/features/super_admin/logging/view_logs_for_ip_spec.rb

```diff
@@ -0,0 +1,78 @@
+describe 'View authentication requests for an IP', type: :feature do
+  let(:user) { create(:user, organisation: user_organisation) }
+  let(:user_organisation) { create(:organisation) }
+  let(:user_location) { create(:location, organisation: user.organisation) }
+  let(:ip_1) { create(:ip, location_id: user_location.id, address: '1.2.3.4') }
+
+  let(:user_2) { create(:user, organisation: user_2_organisation) }
+  let(:user_2_organisation) { create(:organisation) }
+  let(:user_2_location) { create(:location, organisation: user_2.organisation) }
+  let(:ip_2) { create(:ip, location_id: user_2_location.id, address: '1.2.3.5') }
+
+  let(:super_admin) { create(:user, super_admin: true) }
+  let(:super_admin_organisation) { create(:organisation) }
+  let(:super_admin_location) { create(:location, organisation: super_admin.organisation) }
+
+
+  before do
+    Session.create!(
+      start: 3.days.ago,
+      siteIP: ip_1.address,
+    )
+
+    Session.create!(
+      start: 3.days.ago,
+      siteIP: ip_1.address,
+    )
+
+    Session.create!(
+      start: 3.days.ago,
+      siteIP: ip_2.address,
+    )
+
+    Session.create!(
+      start: 3.days.ago,
+      siteIP: ip_2.address,
+    )
+
+    sign_in_user super_admin
+    visit ip_new_logs_search_path
+  end
+
+  context "when searching for an IP from a location" do
+    before do
+      fill_in "IP address", with: ip_1.address
+      click_on "Show logs"
+    end
+
+    it 'displays all found results for that searched IP' do
+      expect(page).to have_content("Found 2 results for \"#{ip_1.address}\"")
+    end
+  end
+
+  context "when looking for an IP from a different location" do
```

Comment:
Why is the " from a different location" important? We aren't searching locations here, just IPs.